guides/v2.2/javascript-dev-guide/widgets/widget_password_strength_indicator.md

```diff
@@ -0,0 +1,123 @@
+---
+group: javascript-developer-guide
+subgroup: 3_Widgets
+title: PasswordStrengthIndicator widget
+---
+
+## Overview {#password_strength_indicator_overview}
+
+Magento password strength indicator {% glossarytooltip f0dcf847-ce21-4b88-8b45-83e1cbf08100 %}widget{% endglossarytooltip %} show the user immediate feedback on the validity and strength of a password while it is being typed.
```

Comment:
The Magento password strength indicator {% glossarytooltip f0dcf847-ce21-4b88-8b45-83e1cbf08100 %}widget{% endglossarytooltip %} provides immediate feedback on the validity and strength of a password while it is being typed.